components/layout_2020/flow/root.rs

```diff
@@ -384,6 +384,21 @@ impl BoxTree {
         let scrollable_overflow = root_fragments
             .iter()
             .fold(PhysicalRect::zero(), |acc, child| {
+                // https://www.w3.org/TR/css-position-3/#fixed-cb
+                // As a result, parts of fixed-positioned boxes that extend outside
+                // the layout viewport/page area cannot be scrolled to and will not print.
```

Comment:
The comment is a bit out of date here now because it only talks about fixed position elements.